packages/libs/client-examples/src/example-contract/describe-account.ts

```diff
@@ -42,9 +25,10 @@ async function getBalanceMain() {
     .setNetworkId('mainnet04')
     .createTransaction();
 
-  const res = await local(tr);
+  const res = await dirtyReady(tr);
 
   console.log(res);
 }
```

Comment:
Shouldn't this be `dirtyRead`? Similar to https://api.chainweb.com/openapi/pact.html#:~:text=In%20a%20blockchain%20environment%20this%20would%20be%20a%20node%2Dlocal%20%E2%80%9Cdirty%20read%E2%80%9D

Reply:
its a typo